src/alpakatest/bin/main.cc

```diff
@@ -69,9 +67,6 @@ int main(int argc, char** argv) {
       datadir = *i;
     } else if (*i == "--transfer") {
       transfer = true;
-    } else if (*i == "--validation") {
-      transfer = true;
-      validation = true;
```

Comment:
Please remove the `validation` from here
https://github.com/cms-patatrack/pixeltrack-standalone/blob/ae72abc417649b0b32eaed854de16980ac454d08/src/alpakatest/bin/main.cc#L121-L122
(it fails to compile otherwise)

It would be good to remove it also from
https://github.com/cms-patatrack/pixeltrack-standalone/blob/ae72abc417649b0b32eaed854de16980ac454d08/src/alpakatest/bin/main.cc#L20
and https://github.com/cms-patatrack/pixeltrack-standalone/blob/ae72abc417649b0b32eaed854de16980ac454d08/src/alpakatest/bin/main.cc#L30